charters/DataOperations/charter.md

```diff
@@ -0,0 +1,89 @@
+# [Data Operations](mailto:data-operations-team@data.humancellatlas.org)
+
+
+## Description
+
+The Data Coordination Platform (DCP) Data Operations Team is responsible for providing the scientific community with effective tools and resources which maximize the value and use of HCA data. This involves identifying the value added for the HCA consumers from various DCP components and the needs and expectations of the consumers, and connecting these communities through useful, fully vetted products and their distribution.
```

Comment:
"effective tools and resources" and "fully vetted products and their distribution" are open to reader interpretation. When I read this, I thought you were taking responsibility for the portals space. This would benefit from qualifying language specifying the types of tools or products you're going to build.

Reply:
@ambrosejcarr @jonahcool @mweiden @parthshahva does 0de0c59 clarify things?

Reply:
Yes, much clearer. Thanks!